IBPSA/Electrical/BaseClasses/PV/BaseClasses/lambertWSimple.mo

```diff
@@ -0,0 +1,21 @@
+within IBPSA.Electrical.BaseClasses.PV.BaseClasses;
+function lambertWSimple
```

Comment:
Extend from `Modelica.Icons...` for it to proper show in OMEdit.

Reply:
done